arduino/strobe/led_strip.h

```diff
@@ -3,9 +3,9 @@
 
 #include "color.h"
 
-// Default the LED_COUNT to 2m of 60 led/m
+// Set the LED count to 540, 18m @ 30/m
```

Comment:
perhaps it's just me but it took me some time to realize the symbols following 540 meant 'strip length by led interval'. Maybe some more english would help, like `18 meter strip @ 30 led/meter`